app/views/items/_new_bid.html.erb

```diff
@@ -1,7 +1,14 @@
-<%= form_for ([item, @bid]) do |f| %>
-  <%= f.number_field :price, min: (item.high_bid+1.00) %>
-  <%= f.submit "Place Bid"  %>
-<% end  %>
+<div id="bid_form">
+    <% if current_user %>
```

Comment:
we should probably put this logic in the item show page, 
so it will read something like
if current user, render the new bid form partial
else, link to log in or create account to bid

unless the item show page is getting crowded...